src/components/ReviewAnswers.js

```diff
@@ -24,20 +24,21 @@ const ReviewAnswers = ({
         Your score is: {score} / {numOfQuestions}
       </p>
       {questions.map(question => (
-        <Card className="mb-4">
+        <Card key={question.id} className="mb-4">
```

Comment:
Great job adding the `key`s to the `map`s, something I missed in my other reviews!